src/pages/develop/cli.md

```diff
@@ -23,6 +23,24 @@ npm i -g @railway/cli
 sh -c "$(curl -sSL https://raw.githubusercontent.com/railwayapp/cli/master/install.sh)"
 ```
 
+**Windows (via Scoop)**
+
+*This should be used incase the npm method doesn't work for you.*
```

Comment:
Change the copy to:

`Use this method if you prefer to interact with Railway using a native Windows binary.`